metadata.json

```diff
@@ -29,6 +29,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">=4.4.0 <5.0.0"
+    },
+    {
+      "name": "jethrocarr/initfact",
+      "version_requirement": ">=0.0.1"
```

Comment:
Makes sense, bound it to 1.x.x - https://github.com/jethrocarr/puppet-yas3fs/commit/c03ad3a031d2802e810eb3cf3533223cbfd18a86